ontology/EBUCore/ebucore.owl

```diff
@@ -4120,12 +4017,11 @@ ec:middleName rdf:type owl:DatatypeProperty ;
 ###  http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#month
 ec:month rdf:type owl:DatatypeProperty ;
          rdfs:range xsd:integer ;
-         rdfs:label "month"@en .
+         rdfs:label "Month"@en .
```

Comment:
Name of month or number of month or both?

Reply:
dfs:range xsd:integer
Doesn't that mean there can only be a number?
Is there a rule, whether to start with 0 or with 1?

Reply:
It is a number. This will be clearer when we in the next pull use the time ontology for this.